joern-cli/frontends/javasrc2cpg/src/test/scala/io/joern/javasrc2cpg/querying/ClassLoaderTypeTests.scala

```diff
@@ -0,0 +1,98 @@
+package io.joern.javasrc2cpg.querying
+
+import io.joern.javasrc2cpg.Config
+import io.joern.javasrc2cpg.testfixtures.JavaSrcCode2CpgFixture
+import io.shiftleft.semanticcpg.language._
+import io.joern.x2cpg.utils.ExternalCommand
+import org.scalatest.Ignore
+
+@Ignore
+class ClassLoaderTypeTests extends JavaSrcCode2CpgFixture {
```

Comment:
In addition to these not active tests, i would like to see a always running test which takes `System.getProperty("java.home")` and feeds it to `withJdkPath`. You can than check for some types which have Java 8 as minimum requirement so that the test can succeed with all expected Java versions. (Java 8 until most recent).